models/comments.models.js

```diff
@@ -20,6 +20,17 @@ module.exports.insertCommentByArticleID = (article_id, username, body) => {
             (%L, %L, %L)
         RETURNING *`, body, article_id, username);
     
+    return db
+        .query(query)
+        .then((result) => result.rows[0]);
+}
+
+module.exports.deleteCommentByID = (comment_id) => {
+    const query = format(`\
+        DELETE FROM comments
+        WHERE comment_id = %L
```

Comment:
You could have made use of the $1 syntax here for avoidance of sql injection and to save you utilising pg-format but it's technically fine.